.github/workflows/build_apk.yml

```
@@ -0,0 +1,70 @@
+
+      - name: Analyze Code
+        run: flutter analyze
+
+      - name: Run Tests (Optional)
+        run: flutter test
+
+      - name: Build APK
+        run: flutter build apk --split-per-abi
+      - name: Check git diff
+        id: git_diff
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)